src/python/grapl-template-generator/grapl_template_generator/python_http_service/create_python_http_service.py

```diff
@@ -0,0 +1,99 @@
+from pathlib import Path
+from typing import Tuple, cast
+
+import toml
+import typer
+from cookiecutter.main import cookiecutter
+from grapl_common.utils.find_grapl_root import find_grapl_root
+from grapl_template_generator.pants_toml_type import PantsToml
+from grapl_template_generator.python_http_service.create_python_http_service_args import (
+    CreatePythonHttpServiceArgs,
+)
+
+
+class PythonHttpServiceTemplateExecutor(object):
+    def __init__(self, args: CreatePythonHttpServiceArgs) -> None:
+        self.project_name = args.project_name
+        self.project_slug = args.project_name.lower().replace(" ", "-")
+        self.pkg_name = args.project_name.lower().replace(" ", "_")
+        self.pants_version = args.pants_version
+        self.pants_python_interpreter_constraints = (
+            args.pants_python_interpreter_constraints
+        )
+        self.lambda_handler = args.lambda_handler
+
+        # TODO: In the future, it might prove more robust to package these
+        # templates as a resources() goal, as opposed to just reading from src/
+        grapl_root = find_grapl_root()
+        assert grapl_root, "Expected to find Grapl root"
+
+        self.grapl_root = grapl_root
+        self.python_src_path = self.grapl_root / "src" / "python"
+        self.template_path = (
+            self.python_src_path
+            / "grapl-template-generator"
+            / "grapl_template_generator"
+            / "grapl-templates"
+            / "grapl-python-service-template"
```

Comment:
(not related to the PR)
i have very mixed feelings about this operator overload
it's too "cute"
cpp does the same thing